multiqc/modules/genomescope2/genomescope2.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse output from genomescope """
+
+from __future__ import print_function
+from collections import OrderedDict
+import logging
+import re, base64
+from multiqc.plots import table
+from multiqc.modules.base_module import BaseMultiqcModule
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+
+class MultiqcModule(BaseMultiqcModule):
+    """Genomescope2 module"""
+
+    def __init__(self):
+
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(
+            name="Genomescope2",
+            anchor="Genomescope2",
+            href="https://github.com/tbenavi1/genomescope2.0",
+            info="Reference-free profiling of polyploid genomes.",
+            doi="https://doi.org/10.1038/s41467-020-14998-3",
+        )
+
+        # Find and load any GENOMESCOPE reports
+        self.summary_data = dict()
+        for f in self.find_log_files("genomescope2/summary"):
+            self.parse_summary_log(f)
+
+        log.info("Found {} summary reports".format(len(self.summary_data)))
```

Comment:
Should be checking that some samples were found before this, and raising a `UserWarning` if not.

I thought that the CI tests should pick this up 🤔 ah well..

Reply:
done

Reply:
Not done, as you raise it but then catch it again. This means that it doesn't bubble up to the code above so it's effectively invisible.

There's generally no need to handle exceptions like this within MultiQC code. If they're unexpected then MultiQC already handles them and continues running.

Reply:
Removed in b889359314e7b8c1916f5eb74b9ee5052cb39e9e ✅